javascript/ql/src/semmle/javascript/frameworks/DateFunctions.qll

```diff
@@ -0,0 +1,61 @@
+private import javascript
+
+private API::Node formatFunction() {
```

Comment:
This is `date-fns` specific, but the predicate name doesn't convey that message.  

How about moving the `date-fns` specific classes/predicates into a `DateFns` module?